scrunch/expressions.py

```diff
@@ -241,10 +242,10 @@ def _parse(node, parent=None):
             else:
                 for _name, _val in fields:
                     if not isinstance(node, ast.UnaryOp) and (
-                            isinstance(_val, ast.BoolOp)
-                            or isinstance(_val, ast.UnaryOp)
-                            or isinstance(_val, ast.Compare)
-                            or isinstance(_val, ast.Call)):
+                            isinstance(_val, ast.BoolOp) or
+                            isinstance(_val, ast.UnaryOp) or
```

Comment:
`isinstance()` supports tupples, you can replace this chain of `or`s with `isinstance(_val, (t1, t2, t3...))`